etc/migration/addSecretColumnToAuthenticationLoginId.cql

```diff
@@ -0,0 +1 @@
+ALTER TABLE AuthenticationLoginId ADD secret varchar;
```

Comment:
This new column should also go in `init.js` (and the file should end in a newline).